### ISSUE TITLE
Body closed during the "Sequential Proxy" feature.

### ISSUE THREAD

**Issue by Alphyron** [luraproject/lura#426]
**Describe the bug**
If you create an endpoint with chaining requests in which the last one is a POST, PUT, or anything other than GET, you receive an error at the last request stating that the body is already closed. So the backend cannot read the body and run without errors.

```krakend_1                 | Error #01: Put "http://endpoint/url": http: invalid Read on closed Body```

**Additional context**
I think the problem might be that Krakend closes the body after the first backend, which for GET requests works without a problem, but not for POST or anything else. So maybe it would help to forward a copy of the body to every single call...
I tried to understand the sequential feature but I did not dig deep enough into the code to see where and how I can change this.
I already had a look at the merging.go file...

**Comment by Alphyron** [comment]
After some reverse engineering and searching, I find a solution to the problem.

```		
buf, _ := ioutil.ReadAll(c.Request.Body)
requestBody := ioutil.NopCloser(bytes.NewBuffer(buf))
copyBody := ioutil.NopCloser(bytes.NewBuffer(buf))
c.Request.Body = copyBody

return &proxy.Request{
	Method:  c.Request.Method,
	Query:   query,
	Body:    requestBody,
	Params:  params,
	Headers: headers,
}
```

You can use this code at the endpoint of the router when creating a new Request.
But I cannot judge if and how much this will influence the performance. With your okay, I will create a pull request.

**Comment by kpacha** [comment]
fixed by #499
